src/wwwroot/template-editor/adapterService.js

```diff
@@ -8,10 +8,11 @@
 
   adapterService.$inject = [
     '$http',
-    '$q'
+    '$q',
+    '$window'
   ];
 
-  function adapterService($http, $q) {
+  function adapterService($http, $q, $window) {
```

Comment:
Is there not a _$localStorage_ or something like that?

Reply:
$window is the service used to access to browser features. I don't found any more in AngularJS api reference.

Reply:
You are right, I saw an npm package for that, so, it seems that there is nothing out of the box. Keep it as it is right now. Thanks!